bootstrap/sw-toolbox-setup.js

```diff
@@ -21,10 +21,28 @@
     var strategy = global.params.get('defaultCacheStrategy');
     global.toolbox.router.default = global.toolbox[strategy] || global[strategy];
   }
-  
-  if (global.params.has('precache')) {
-    global.toolbox.precache(global.params.get('precache'));
+
+  var precachePromise;
+  // When precacheHash is present inside the cacheConfigFile JSON, its a signal that instead of
```

Comment:
A minor comment mistake here: I believe that you mean the precacheHash is present in the querystring options, rather than in the JSON.